docs/manual/installation/move-contao.de.md

```diff
@@ -77,18 +77,28 @@ Gib dein Passwort ein, wenn du danach gefragt wirst.
 {{< /tabs >}}
 
 
+## Einstellungen notieren
+Deine Einstellungen unter `System / Einstellungen` oder im Seitenroot notierst du dir am besten und trägst sie später erneut ein.
```

Comment:
That is not necessary if you also copy the `system/config/localconfig.php`

Reply:
I find it misleading if there is said "alte Konfigurationen unter system/config", as I did not use any "old" configuratiuons as fas as I know.

Reply:
Just saying. You can of course improve it ;)

Reply:
I hope my changes make it clear now